.gitignore

```diff
@@ -12,3 +12,6 @@ node_modules/
 
 # OS Specific
 .DS_Store
+.vscode/launch.json
+*.code-workspace
+.vscode/launch.json
```

Comment:
These are IDE entries, so they should go under the IDE comment. Also, `.vscode/launch.json` is here twice. It should probably just be `.vscode/` to the entire directory is ignored.